opal-gwt-client/src/main/java/org/obiba/opal/web/gwt/app/client/magma/view/ValuesTableView.ui.xml

```diff
@@ -95,7 +95,7 @@
 
       <o:OpalSimplePager ui:field="pager" showTotalWhenNoPager="true" location="RIGHT" addStyleNames="pull-right"/>
     </g:FlowPanel>
-
+    <g:FlowPanel ui:field="warningsPanel" visible="false"/>
```

Comment:
if you use this for notification warning...what happen when you come back to the same screen (without refresh)?
check http://www.gwtproject.org/javadoc/latest/com/google/gwt/user/client/ui/NotificationMole.html